### DISCUSSION TITLE
Default Pileup Color Scheme

### DISCUSSION THREAD

**Discussion by jrodiger-lifemine** [GMOD/jbrowse-components#3417] [Q&A] [answered]
Is it possible to configure the default pileup color scheme for a track or all alignment tracks? The documentation mentions that by default reads are colored by strand but this doesn't seem to be the case unless you select the strand color scheme through the UI.

https://jbrowse.org/jb2/docs/user_guides/alignments_track/#pileup-visualization

**Comment by cmdcolin** [comment, marked as answer]
There is actually no way to default configure the color scheme, it is currently only enabled via "session state" (e.g. sharing a session with this setting enabled for example, or setting up a defaultSession in the config) 

Also the docs should be updated there, indeed those docs are from the very early days, it was changed from color by strand by default to the plain grey shortly after that. Are you interested in turning strand on by default? I am interested in learning e.g. biological cases when strand is most appropriate.

We can try to add this to configuration though!

**Reply by jrodiger-lifemine** [reply]
Gotcha our users requested having the coloring by strand as default but we can do this through the UI for now. Thank you!
